pipelines/ApPipe.yaml

```diff
@@ -17,17 +17,26 @@ tasks:
   imageDifference:
     class: lsst.pipe.tasks.imageDifference.ImageDifferenceTask
     config:
-      doWriteWarpedExp: True
-      coaddName: goodSeeing
-      getTemplate.coaddName: goodSeeing
+      doWriteWarpedExp: True             # Required for diaPipe
+      # 'goodSeeing' expected type for LSST processing; task default 'deep' for Gen 2 pipelines
+      coaddName: goodSeeing              # Redundant line for Gen 2 compatibility
+      getTemplate.coaddName: goodSeeing  # Redundant line for Gen 2 compatibility
```

Comment:
I'm confused about the "gen2 compatibility" statement here: this file is only used in gen3. The comment above says that "goodSeeing" is the correct name here, but the "redundant" line suggests these are unnecessary?

Reply:
`coaddName` and `getTemplate.coaddName` are used by Gen 2, and the config requires that they and `connections.coaddName` all be consistent with each other. "Redundant" in the sense that ideally the coadd name would only need to be specified in one place.